index.html

```diff
@@ -337,7 +341,45 @@ <h2>Contact me</h2>
                 </section>
             </div>
         </div>
+        <!-- Modal -->
+        <div class="Modalcontainer vanish">
+            <div class="Modalcont">
+                <div class="Modalhead">
+                    <h2 class="Modalh"></h2>
+                    <span class="cross">&times;</span>
+                </div>
+                <div class="Modallang">
+                    <p class="Modalcan"></p>
+                    <ul class="Modallist">
+                        <li class="Modalrole"></li>
+                        <li class="Modalyear"></li>
+                    </ul>
+                </div>
+                <div class="Modalimage"><img src="#" class="ModalImg" alt=""></div>
+                <div class="Modalwp">
+                    <p class="Modalpara">
+                    </p>
+                    <div class="Modalw">
+                        <ul class="ModalWorks">
+
+                        </ul>
+                        <ul class="ModalWorks2">
+                            <li class="tag">Github</li>
+                            <li class="tag">Ruby</li>
+                            <li class="tag">Bootstrap</li>
+                        </ul>
+
+                        <div class="Modalbuttons">
+                            <button class="Modalbtn1">See live <img src="./images/Vector.svg" alt=""></button>
+                            <button class="Modalbtn1">See sources <img src="./images/Icon.svg" alt=""></button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] Following the project requirements, The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. Check out the requirements below 👇 :

![](https://user-images.githubusercontent.com/77190203/157649553-844d7570-0411-4439-a7d8-7ddd9a4de306.png)